presto-mysql/src/main/java/com/facebook/presto/plugin/mysql/MySqlClient.java

```diff
@@ -176,4 +185,44 @@ protected String toSqlType(Type type)
 
         return super.toSqlType(type);
     }
+
+    @Override
+    public void createTable(ConnectorTableMetadata tableMetadata)
+    {
+        try {
+            createTable(tableMetadata, tableMetadata.getTable().getTableName());
+        }
+        catch (SQLException e) {
+            if (DUPLICATE_TABLE_SQLSTATE.equals(e.getSQLState())) {
+                throw new PrestoException(ALREADY_EXISTS, e);
+            }
+            else {
+                throw new PrestoException(JDBC_ERROR, e);
+            }
+        }
+    }
+
+    @Override
+    public void renameColumn(JdbcTableHandle handle, JdbcColumnHandle jdbcColumn, String target)
+    {
+        try (Connection connection = connectionFactory.openConnection()) {
+            DatabaseMetaData metadata = connection.getMetaData();
+            if (metadata.storesUpperCaseIdentifiers()) {
+                target = target.toUpperCase(ENGLISH);
+            }
+            String sql = format(
+                    "ALTER TABLE %s RENAME COLUMN %s TO %s",
+                    quoted(handle.getCatalogName(), handle.getSchemaName(), handle.getTableName()),
+                    quoted(jdbcColumn.getColumnName()),
+                    quoted(target));
+            execute(connection, sql);
+        }
+        catch (SQLException e) {
+            // MySQL versions earlier than 8 do not support the above RENAME COLUMN syntax
+            if (UNSUPPORTED_SYNTAX.equals(e.getSQLState())) {
+                throw new PrestoException(NOT_SUPPORTED, format("Rename column not supported in catalog '%s'", handle.getCatalogName()), e);
```

Comment:
It's preferred to use a colon in an error message when there is only one variable part:
```java
throw new PrestoException(NOT_SUPPORTED, "Rename column not supported in catalog: " + handle.getCatalogName(), e);